src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -531,15 +533,20 @@ private void buildCcjEvent(EventHistory.EventHistoryBuilder builder, CaseData ca
             buildJudgmentByAdmissionEventDetails(builder, caseData);
 
             String miscTextRequested = RPA_REASON_JUDGMENT_BY_ADMISSION;
+            String detailsTextRequested = RPA_REASON_JUDGMENT_BY_ADMISSION;
             if (featureToggleService.isJOLiveFeedActive()) {
-                miscTextRequested = RPA_RECORD_JUDGMENT;
+                miscTextRequested = RECORD_JUDGMENT;
+                detailsTextRequested = RPA_RECORD_JUDGMENT_REASON;
+                if (caseData.getCcdState() == CaseState.PROCEEDS_IN_HERITAGE_SYSTEM) {
```

Comment:
so if i run it today for exmaple it's `RPA_RECORD_JUDGMENT_REASON`
in the future it might change to `RPA_RECORD_JUDGMENT_OFFLINE` if it takes offline.

It looks like this can be updated.. not sure if this is right? What do you think @Azam-Hmcts

Reply:
should we have a second 999 event to signify it gets taken offline?